addons/mail/tests/test_mail_render.py

```diff
@@ -374,8 +374,7 @@ def test_template_rendering_various(self):
                 src, partner._name, partner.ids, engine=engine,
             )[partner.id]
             self.assertEqual(expected, str(result))
-        src = """<p>
-<t t-set="line_statement_variable" t-value="3" />
+        src = """<p><t t-set="line_statement_variable" t-value="3" />
```

Comment:
This should not change. It indicates there is an issue when rendering about line breaks.